.github/workflows/test-matrix.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Do we have a plan to support all platforms?
Workspaces JS should with on macOS CI soon.

Reply:
This test suite doesn't do anything platform specific, the e2e do.